Webdev4.html

```diff
@@ -0,0 +1,163 @@
+                subdomains: 'abcd',
+                maxZoom: 19
+            }).addTo(map);
+
+            $.getJSON("../data/duluth_precincts_WGS84.geojson",function(data){
+
+              L.geoJson(data,{
+                  style: function (feature) {
+
+                      var votePcts = calculatePartyPct(feature);
+
+                      var fill;
+
+                      var colors = ['#ca0020','#f4a582','#f7f7f7','#92c5de','#0571b0'];
+                      console.log('votePcts', votePcts);
+
+                      if (votePcts.rep >= 40) {
+                          fill = colors[0];
+                          console.log('rep40');
+                      }
+                      else if (votePcts.rep >= 35) {
+                          fill = colors[1];
+                          console.log('rep30');
+                      }
+                      else if (votePcts.rep >= 30 && votePcts.dem <= 59) {
+                          fill = colors[2];
+                          console.log('mid');
+                      }
+                      else if (votePcts.dem >= 60 && votePcts.dem <= 65) {
+                          fill = colors[3];
+                          console.log('dem65');
+                      }
+                      else {
+                          fill = colors[4];
+                          console.log('else');
+                      }
+                      return {
+                          color: '#636363',
+                          weight: 2,
+                          fillColor: fill,
+                          fillOpacity: 0.7
+                      }
+                  },
+                  onEachFeature: function (feature, layer) {
+                      var votePcts = calculatePartyPct(feature)
+                      layer.bindPopup(
+                          "<h3>" + feature.properties.PCTNAME + "</h3>" +
+                          "<p>Democrat Vote: <span class='dem'>" + votePcts.dem + "%</span></p>" +
+                          "<p>Republican Vote: <span class='rep'>" + votePcts.rep + "%</span></p>"
+                      );
+                  }
+              }).addTo(map);
+          });
+          function calculatePartyPct(feature) {
+              var demVote = feature.properties.USPRSDFL;
+              var repVote = feature.properties.USPRSR;
+              var totalVote = feature.properties.USPRSTOTAL;
+              var demPct = Math.round(demVote/totalVote*100);
+              var repPct = Math.round(repVote/totalVote*100);
+              return {
+                  rep: repPct,
+                  dem: demPct
+              };
+          }
+      </script>
+  <script>
+
+      var map = L.map('map').setView([46.7867, -92.1005], 11);
+
+      L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', { attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>' }).addTo(map);
```

Comment:
One other issue is that you added the map and basemap (`L.tileLayer`) multiple times for some reason.